rs/protobuf/def/state/queues/v1/queues.proto

```diff
@@ -39,13 +39,11 @@ message StreamFlags {
 }
 
 message Stream {
+  reserved 3, 4, 6;
+  reserved "signals_begin", "signals", "deprecated_reject_signals";
```

Comment:
Nit: I would put this at the end. It's definitely not the most important aspect of this message definition.

I suppose the other messages have it at the top because we often deprecate field 1 and then put the `reserved` entries where the field used to be. But that's not the most useful criterion.

Reply:
Moved.